Makefile

```diff
@@ -1,10 +1,13 @@
 .PHONY: collector
 
 collector: check-env generate-key
+	@test -f /usr/bin/qemu-aarch64-static
 	@cp -R ds-collector/ collector
 	@rm -f collector/collect-info
-	@cd collector ; docker run --rm -v $$PWD:/volume -w /volume -t clux/muslrust rustc --target x86_64-unknown-linux-musl rust-commands/*.rs ; cd -
-	@test -f collector/collect-info
+	@cd collector ; docker run --rm --platform linux/arm64 -v /usr/bin/qemu-aarch64-static:/usr/bin/qemu-aarch64-static -v $$PWD:/volume -w /volume -t clux/muslrust rustc --target aarch64-unknown-linux-musl rust-commands/*.rs ; mv collect-info collect-info.aarch64-unknown-linux-musl ; cd -
```

Comment:
the `cd -` at the end just undoes the `cd collector` from the beginning?

Reply:
intentional.